tests/test_trimming.py

```diff
@@ -0,0 +1,61 @@
+from lingrex.trimming import (
+        apply_trim, revert, get_skeleton, consecutive_gaps, gap_profile,
+        trim_by_gap,trim_by_core, trim_random, subsequence_of
+        )
+import random
+random.seed(1234)
```

Comment:
That's a dangerous thing to do, because this changes global state. So if some other test module sets `random.seed` differently, the order of import will determine which setting wins.

So either do that in `conftest.py` to make it more transparent for other tests, or - maybe better - mock `random` for the test in question.